### PR TITLE
Fix React Native intro link in readme

### PR DESCRIPTION
**Pull request by markdalgleish**
First PR! 👋 Sorry it's not more exciting.

### CODE REVIEW

**Comment by neilthawani** [comment]
Seems ok.

**Comment by jmahhh** [comment]
LGTM 420 BLAZE IT

**Comment by platocrat** [comment]
lgtm 420 blaze it

**Comment by ok1a** [comment]
[lgtm 420 blaze it](https://twitter.com/mikedemarais/status/1437595522114768904)

**Comment by kcherukuri** [comment]
lgtm 420 blaze it

**Comment by jxom** [comment]
lgtm 420 blaze it

**Review by mikedemarais — APPROVED** [review]
lgtm 420 blaze it

**Review by skylarbarrera — APPROVED** [review]
:shipit:

**Comment by christianbaroni** [comment]
lgtm 420 blaze it

**Comment by Alexangelj** [comment]
lgtm 420 blaze it

**Comment by scotato** [comment]
lgtm 420 blaze it

**Comment by garythung** [comment]
lgtm 420 blaze it

**Comment by kunalkundaje** [comment]
[lgtm 420 blaze it](https://twitter.com/mikedemarais/status/1437595522114768904?s=21)

**Comment by thdxr** [comment]
Lgtm 420 blaze it

**Comment by lochie** [comment]
# ļ̵̜̼̭̊̀̂g̴̛̳̞̹̮͚͐̎͠t̸̨͖̯̗̋͛͝m̶̧͎̹͆̉͛͗͝ ̷̥̪̈́̄͛4̷̭͖͉̇2̷̝͘0̷̡̙̣͂̍̈́̀͝ ̷̮̟̃̆̋̅͝b̴̰̺̊̿͂͘͠l̵̢̘͓̱̂̉͜a̸͇̤͈͛́̆̿z̷̭̙̺̞̋͘e̸̢̼̹̓̈̃͆͜ ̶̹̩̆̏̋ī̶̱̖̜͜t̴͉̐̆̏̈̓

**Comment by Anish-Agnihotri** [comment]
lgtm, 420 blaze it

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
 ## Developing
 
 If you are new to React Native, this is a helpful introduction:
-https://facebook.github.io/react-native/docs/getting-started.html
+https://reactnative.dev/docs/getting-started
```
Comment (nandorojo) [inline README.md:59]: Please make the letters rainbow colored for clarity. Besides that, LGTM

**Comment by edennington** [comment]
lgtm 420 blaze it

File: README.md (hunk 1 above)
Comment (TheMightyPenguin) [reply README.md:59]: @markdalgleish either add an emoji here or no approval from me

**Review by malonehedges — APPROVED** [review]
lgtm 420 blaze it

**Comment by vasilenka** [comment]
lgtm 420 blaze it

**Comment by bford21** [comment]
lgtm 420 blaze it

**Comment by m1guelpf** [comment]
LGTM 420 BLAZE IT

**Comment by anshulporwal26** [comment]
LGTM 420 BLAZE IT

**Comment by sggts04** [comment]
lgtm 420 blaze it

**Comment by gorhom** [comment]
LGTM, ship it 🚀

**Comment by rozenmd** [comment]
LGTM mate 👍

File: README.md (hunk 1 above)
Comment (whenmoon) [reply README.md:59]: gm

**Comment by kinduff** [comment]
lgtm 420 blaze it

File: README.md (hunk 1 above)
Comment (PeerRich) [reply README.md:59]: gm

**Comment by PeerRich** [comment]
lgtm 420 blaze it

**Comment by shellscape** [comment]
Here for the ratio

**Comment by pedrouid** [comment]
lgtm 420 blaze it

**Comment by benbasche** [comment]
lgtm 420 blaze it

File: README.md (hunk 1 above)
Comment (sui414) [reply README.md:59]: gm

**Pull request merged**